### ISSUE TITLE
CHANGELOG

### ISSUE THREAD

**Issue by barberj** [tshaddix/webext-redux#194]
Is everything backward compatible? 

Can you add a CHANGELOG detailing changes so earlier versions can upgrade accordingly?

**Comment by tshaddix** [comment]
@barberj Yep - other than changing the name of `react-chrome-redux` to `webext-redux`, everything should be backwards compatible. 

A changelog is a good idea. Should be relatively straight-forward as everything is documents in https://github.com/tshaddix/webext-redux/releases

**Comment by barberj** [comment]
awesome. just upgraded in a project
